wls-basisdaten-service/src/test/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/rest/wahltage/WahltageControllerIntegrationTest.java

```diff
@@ -0,0 +1,218 @@
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.domain.WahlvorschlagRepository;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.eai.aou.model.WahltagDTO;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.services.wahltag.WahltagModelMapper;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.utils.Authorities;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.rest.model.WlsExceptionCategory;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.rest.model.WlsExceptionDTO;
+import de.muenchen.oss.wahllokalsystem.wls.common.testing.SecurityUtils;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest(classes = MicroServiceApplication.class, webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+@AutoConfigureWireMock
+@ActiveProfiles(profiles = { SPRING_TEST_PROFILE, SPRING_NO_SECURITY_PROFILE })
+public class WahltageControllerIntegrationTest {
+
+    @Value("${service.info.oid}")
+    String serviceID;
+
+    @Autowired
+    MockMvc api;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Autowired
+    WahltageDTOMapper dtoMapper;
+
+    @Autowired
+    WahltagModelMapper modelMapper;
+
+    @Autowired
+    WahltageClientMapper wahltageClientMapper;
+
+    @Autowired
+    WahltagRepository wahltagRepository;
+
+    @Autowired
+    WahlvorschlagRepository wahlvorschlagRepository;
+
+    @Autowired
+    KandidatRepository kandidatRepository;
+
+    @AfterEach
+    void tearDown() {
+        SecurityUtils.runWith(Authorities.ALL_AUTHORITIES_DELETE_WAHLTAGE);
+        wahltagRepository.deleteAll();
+    }
+
+    @BeforeEach
+    void setup() {
+        WireMock.resetAllRequests();
+    }
+
+    @Nested
+    class GetWahltage {
+
+        @Test
+        void loadedFromExternal() throws Exception {
+
+            String requestDate = LocalDate.now().minusMonths(3).toString();
```

Comment:
- Leerziele entfernt
- Umstellung auf final

```suggestion
        void loadedFromExternal() throws Exception {
            val requestDate = LocalDate.now().minusMonths(3).toString();
```

Reply:
resolved 6385ce3f0f35cff9179b2315f5b8f6be4d73a968

Reply:
Leerzeile ist noch da